### PR TITLE
Feature/time sheets

### PR DESCRIPTION
**Pull request by mani9694**

### CODE REVIEW

File: app/Http/Controllers/TimesheetsController.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Show the application dashboard.
+     *
```
Comment (gopibabus) [inline app/Http/Controllers/TimesheetsController.php:19]: Please remove this file. Requested changes should be part of existing page.

File: resources/views/timesheets.blade.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container jumbotron">
+        <h1 class="display-4">Welcome to Timesheets</h1>
+@endsection
```
Comment (gopibabus) [inline resources/views/timesheets.blade.php:6]: Please remove this file. View is not required at this point.

File: routes/web.php [hunk 3]
```diff
@@ -20,3 +20,4 @@
 Auth::routes();
 
 Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+Route::get('/time-sheets', [App\Http\Controllers\TimesheetsController::class, 'index']);
```
Comment (gopibabus) [inline routes/web.php:23]: Remove this route. Route is not required at this point.

**Review by gopibabus — CHANGES_REQUESTED** [review]

**Comment by mani9694** [comment]
Deleted not need files and updated the PR.
This is how it looks in my local.
Screen Shot 2020-12-06 at 6 59 39 PM

File: resources/views/layouts/app.blade.php [hunk 4]
```diff
@@ -27,6 +27,9 @@
                 <a class="navbar-brand" href="{{ url('/') }}">
                     {{ config('app.name', 'Laravel') }}
                 </a>
+                <a class="navbar-brand" href="{{ url('/') }}">
+                    {{ 'Time sheets' }}
+                </a>
```
Comment (gopibabus) [inline resources/views/layouts/app.blade.php]: @mani9694 we should restrict this menu item for only logged in users. I am applying that fiix.

**Pull request merged**